rules/windows/defense_evasion_disabling_windows_logs.toml

```diff
@@ -0,0 +1,49 @@
+[metadata]
+creation_date = "2021/05/06"
+maturity = "development"
+updated_date = "2021/05/06"
+
+[rule]
+author = ["Elastic","Ivan Ninichuck","Austin Songer"]
+description = """
+Identifies attempts to disable EventLog and logman. This is often done by attackers in an attempt to evade detection on
+a system.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Disable Windows Event and Security Logs"
+references = ["https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/logman"]
+risk_score = 21
+rule_id = "4de76544-f0e5-486a-8f84-eae0b6063cdc"
+severity = "low"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Defense Evasion"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
```

Comment:
nice! the current query will return unrelated events (i.e. `cmd.exe /c delete file.txt`, you can adjust it as shown below:

```
process where event.type in ("start", "process_started") and
     ((process.name:"logman.exe" or process.pe.original_file_name == "Logman.exe") and 
           process.args : "EventLog-*" and process.args : ("stop", "delete")) or         
     ((process.name : ("pwsh.exe", "powershell.exe", "powershell_ise.exe") or process.pe.original_file_name in 
 ("pwsh.exe", "powershell.exe", "powershell_ise.exe")) and 
			process.args : "Set-Service" and process.args: "EventLog" and process.args : "Disabled")  or 
		  ((process.name:"auditpol.exe" or process.pe.original_file_name == "AUDITPOL.EXE") and process.args : "/success:disable")
```

Also there more methods to achieve the same that can be covered with same rule (e.g. auditpol):

https://svch0st.medium.com/event-log-tampering-part-1-disrupting-the-eventlog-service-8d4b7d67335c